torchao/quantization/quant_primitives.py

```diff
@@ -267,6 +267,11 @@ def _quantize_affine_no_dtype_cast(
     if zero_point is not None:
         zero_point = zero_point.view(shape_after_reduction)
 
+    if zero_point_domain is None:
+       quant = torch.clamp(input * scale.reciprocal(), quant_min, quant_max)
```

Comment:
how do we know if we should do `input * scale.reciprocal()` v.s. `input / scale` v.s. `input * (1.0 / scale)`...

Reply:
Way back when Nikita told me that this is the fastest way to do it, I think we should update the others as well but probs in a separate PR since this is already pretty sprawling